docs/deployment.md

```diff
@@ -59,3 +59,38 @@ $ heroku addons:open scheduler
 At this point you should have the Heroku Scheduler Dashboard open. Click "Add new job" and enter the same command as you ran previously (e.g. `renovate [your/repo]`). Adjust the frequency to hourly if you prefer, then click Save.
 
 You can run `heroku logs` to check execution logs. Consider adjusting the scripts log level if you have problems (info -> verbose -> debug -> silly).
+
+## Docker
+
+Renovate can also be run as a Docker container.
+
+### Using docker-compose
```

Comment:
I find it a bit confusing to mention docker-composer here even though there are no images composed